ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/vars/CentOS.yml

```diff
@@ -9,7 +9,6 @@ Build_Tool_Packages:
   - autoconf
   - bind-utils
   - bison
-  - bzip2
```

Comment:
Why remove this if its in the redhat playbook?

Reply:
I'll undo this - was initially because lbzip2 was in the CentOS playbook